src/Sarif/Baseline/V2/V2ResultMatcher.cs

```diff
@@ -1,11 +1,10 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+using Microsoft.CodeAnalysis.Sarif.Baseline.ResultMatching;
```

Comment:
Nit: I believe the SDK's convention is "System" namespaces first.

Reply:
Fixed. Wish VS would insert them following our formatting guidelines. :/